.travis.yml

```diff
@@ -1,87 +1,50 @@
-language: python
-
-python:
-    - 3.7
+language: c
```

Comment:
This confuses me - why set the language to C when it's all Python code? 

(To be clear I'm not objecting if this is part of the recommended Travis setup. I'm just saying I don't understand it.)

Reply:
@mperrin , this is "inherited" from the Astropy project. Python is not available natively on all OSes in Travis, so Astropy uses the `conda` version of Python via `ci-helpers`. Therefore, the "language" does not really matter as long as you grab the conda's Python via `ci-helpers`.